docs/archive/build-apps/apps.md

```diff
@@ -11,85 +11,85 @@ Payment-based applications have a primary purpose of transferring funds from one
 
-<center>![Applications that Delegate Spending Authority](../imgs/app-7.jpg)</center>
+<center>![Applications that Delegate Spending Authority](../../imgs/app-7.jpg)</center>
 
 # Unique Global or User-Based Variable Applications
 These types of applications require customizable global and local variables to keep track of application and user state.  For example, a voting application requires a list of candidates to vote for, with their current vote total and users who cast a vote prevent double-voting. In this example, the vote total is stored globally and a user is marked locally as having voted. An auction application may list items globally and individual bids may be stored locally. 
 
 Stateful Smart Contracts are Algorand’s technology for creating applications with global or user-based state. Algorand provides two types of layer-1 smart contracts, stateless (see Escrow Account-Based Applications or Delegating Authority Applications) and stateful. Stateful Smart Contracts are logic programs written in an assembly-like language called the Transaction Execution Approval Language (TEAL), just like stateless smart contracts. However, unlike their stateless counterparts, stateful smart contracts live on the Algorand blockchain and can store values globally or locally in the balance record of a specific account. Stateful smart contracts, also called Applications, can be called with the use of a special type of transaction. The logic in the program then decides on how to process the specific call (e.g. change or update program state) and whether or not to pass or fail the transaction. 
 
-In the case of a voting application, a vote is just a specialized transaction where the stateful smart contract increments the vote count for the candidate of choice by the user and also marks the user’s account as having voted (stored as a value in the user’s local state). For more information see the [Stateful Smart Contract](../features/asc1/stateful/index.md) guide.
+In the case of a voting application, a vote is just a specialized transaction where the stateful smart contract increments the vote count for the candidate of choice by the user and also marks the user’s account as having voted (stored as a value in the user’s local state). For more information see the [Stateful Smart Contract](../../get-details/dapps/pyteal/smart-contracts/stateful) guide.
```

Comment:
@avesda  is it possible to make this /stateful/index.md?

Reply:
It seems like mkdocs automatically detects the index.md and launches it. If I add index.md in the url I will get 404.